code/modules/clothing/gloves/color.dm

```diff
@@ -221,7 +219,6 @@
 	siemens_coefficient = 0.3
 	permeability_coefficient = 0.01
 	clothing_traits = list(TRAIT_QUICK_CARRY)
-	transfer_prints = TRUE
```

Comment:
So latex gloves will have to not cover hands? That doesn't make sense--something that transfers prints but also covers hands is totally reasonable. They should have a trait.

Reply:
I was concerned over that before too.

Reply:
is there a way to give an item a trait without needing to do `ADD_TRAIT()` in an `Initialize()` proc? Like, do they have a list of some sort?
If you don't know that's fine, I can just do the initialize method if there isn't.

Reply:
I made a flag and put it on all the gloves that previously allowed fingerprints to go through.

Reply:
in real life nitrile gloves are actually so thing that they will transfer impressions of fingerprints.
latex gloves will not.

Reply:
Alright, it's a trait now instead of a flag.